manager_task_status_pub_rate: 1
mission_tasks:
- jiangshi
- droppers
startup_delay: 0
tasks:
  droppers:
    drop_depth: -2
    manager_info:
      outcome_not_found: skip
      outcome_timed_out: skip
      timeout_secs: 0
    prior:
    - -11.6
    - -2.53
    - -1.0
    retrace_hit_count: 1
    retrace_timeout: 15
    seconds_in_position: 2
    xy_dist_thresh_app: 0.5
    xy_dist_thresh_drop: 0.1
  jiangshi:
    approach_carrot_dist: 2
    approach_stop_dist: 2.5
    height2dist_intercept: 6.672
    height2dist_slope: -0.0125
    manager_info:
      outcome_not_found: skip
      outcome_timed_out: skip
      timeout_secs: 0
    new_dv_check_rate: 30
    prior:
    - -7.25
    - -0.73
    - -1.5
    retrace_back_in_time: 5
    retrace_hit_count: 1
    retrace_timeout: 10
    seconds_in_position: 2
    slay_carrot: 0.5
  prequal:
    manager_info:
      outcome_not_found: skip
      outcome_timed_out: skip
      timeout_secs: 0
  start_gate:
    dist_behind_gate: 1.0
    do_style: true
    manager_info:
      outcome_not_found: skip
      outcome_timed_out: uh_oh
      timeout_secs: 0
    prior:
    - -14.73
    - 4.23
    - -2.0
    replan_thresh: 1.0
    spin_carrot_rads: 0.7
    style_dist: 2.0
    third_leg_adjustment: 0.5
    three_leg_depth: -0.4
  startup:
    depth: -1.5
    dive_time: 1
    manager_info:
      timeout_secs: 1
  torpedos:
    manager_info:
      outcome_not_found: skip
      outcome_timed_out: skip
      timeout_secs: 0
    prior:
    - -15.7
    - -3.45
    - -2.0
using_sim_params: true
